lib/vdsm/storage/hsm.py

```diff
@@ -34,6 +34,7 @@
 from vdsm.common.time import monotonic_time
```

Comment:
For the commit message:
> - exclusive lock on the base image

Remove `base`, image is the disk including base and top.